### ISSUE TITLE
Killed

### ISSUE THREAD

**Issue by 3DTalker** [yerfor/Real3DPortrait#19]
Hey,
I was trying to reproduce your results and every time my process got killed after rendering frames.
I've tried both CLI and Gradio.

Here is the output I'm getting:
~/Real3DPortrait$ python inference/real3d_infer.py --src_img data/raw/examples/WIN_20240216_15_02_39_Pro.jpg --drv_aud data/raw/examples/Base_1.mp3 --out_name output.mp4 \
> 
| WARN: egs/egs_bases/audio2motion/vae.yaml not exist.
| WARN: checkpoints/240210_real3dportrait_orig/th1kh_512_audio2motion/base.yaml not exist.
| load 'model' from 'checkpoints/240210_real3dportrait_orig/audio2secc_vae/model_ckpt_steps_400000.ckpt', strict=True
| WARN: checkpoints/240210_real3dportrait_orig/secc2plane_torso_orig/secc_img2plane_orig.yaml not exist.
INFO:torch.distributed.nn.jit.instantiator:Created a temporary directory at /tmp/tmpmkvu3775
INFO:torch.distributed.nn.jit.instantiator:Writing /tmp/tmpmkvu3775/_remote_module_non_scriptable.py
| load 'model' from 'checkpoints/240210_real3dportrait_orig/secc2plane_torso_orig/model_ckpt_steps_100000.ckpt', strict=True
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1708355852.315639   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355852.368359   30088 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
W0000 00:00:1708355852.368894   30029 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
face area percent 0.04235999857083783 smaller than threshold 0.2, crop the input image...
I0000 00:00:1708355852.609835   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355852.650467   30093 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
W0000 00:00:1708355852.650889   30029 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
Extracted wav file (16khz) from data/raw/examples/Base_1.mp3 to data/raw/examples/Base_1_16k.wav.
Loading the HuBERT Model...
Loading the Wav2Vec2 Processor...
I0000 00:00:1708355868.580258   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355868.626693   30153 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
combining backgrounds...: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  1.44it/s]
I0000 00:00:1708355870.297166   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355870.346573   30176 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
W0000 00:00:1708355870.347007   30029 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
batch 0 | loss_lan: 1.02, loss_reg_id: 1.91,loss_reg_exp: 2.91
| To extract pose from data/raw/examples/May_5s.mp4
lms_2d file not found, try to extract it from video... data/raw/examples/May_5s_lms.npy
I0000 00:00:1708355871.993587   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355872.035644   30191 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
W0000 00:00:1708355872.036068   30029 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
I0000 00:00:1708355872.059524   30029 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1708355872.100376   30196 gl_context.cc:344] GL version: 3.2 (OpenGL ES 3.2 NVIDIA 535.104.12), renderer: Tesla T4/PCIe/SSE2
W0000 00:00:1708355872.100757   30029 face_landmarker_graph.cc:174] Sets FaceBlendshapesGraph acceleration to xnnpack by default.
| Extracted pose from data/raw/examples/May_5s.mp4
{'drv_audio_name': 'data/raw/examples/Base_1.mp3', 'src_image_name': 'infer_out/tmp/cropped_src_img.png', 'a2m_ckpt': 'checkpoints/240210_real3dportrait_orig/audio2secc_vae', 'head_ckpt': '', 'torso_ckpt': 'checkpoints/240210_real3dportrait_orig/secc2plane_torso_orig', 'bg_image_name': '', 'drv_pose_name': 'data/raw/examples/May_5s.mp4', 'blink_mode': 'period', 'temperature': 0.2, 'mouth_amp': 0.45, 'out_name': 'output.mp4', 'out_mode': 'concat_debug', 'map_to_init_pose': 'True', 'head_torso_threshold': None, 'seed': None, 'min_face_area_percent': 0.2, 'low_memory_usage': False}
| audio-to-motion finished
rendering drv secc:   0%|                                                                                                                                          | 0/6 [00:00<?, ?it/s]create rasterizer on device cuda:0
rendering drv secc: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:09<00:00,  1.54s/it]
blinking secc: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 252/252 [00:01<00:00, 203.69it/s]
Real3D-Portrait is rendering frames:   0%|                                                                                                                       | 0/252 [00:00<?, ?it/s]Setting up PyTorch plugin "bias_act_plugin"... Done.
Setting up PyTorch plugin "upfirdn2d_plugin"... Done.
Real3D-Portrait is rendering frames: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 252/252 [01:32<00:00,  2.72it/s]
Killed

**Comment by wangnan0610** [comment]
hello~你怎么解决这个错误的呢

**Comment by thanhhung0112** [comment]
hello, can you share how to resolve this issue?, I also encountered this when running on Colab

**Comment by GeneralWhite** [comment]
+1

**Comment by joytsay** [comment]
adding `--out_mode final` to `python inference/real3d_infer.py`
solved the issue
